examples/data-objects/prosemirror/src/fluidCollabManager.ts

```diff
@@ -413,7 +418,7 @@ export class FluidCollabManager extends EventEmitter implements IRichTextEditor
                 }
 
                 case "addMark": {
-                    const attrs = stepAsJson.mark.attrs || true;
```

Comment:
This looks strange - is "true" really a valid value for attrs?  Looks like it wants an object dictionary normally?